contracts/governance/Cargo.toml

```
@@ -32,5 +32,6 @@ shade-protocol = { version = "0.1.0", path = "../../packages/shade_protocol" }
 schemars = "0.7"
 serde = { version = "1.0.103", default-features = false, features = ["derive"] }
 snafu = { version = "0.6.3" }
+serde_json = { version = "1.0.67"}
```

Is this import used?

---

I'm using it in tests here: https://github.com/securesecrets/shade/pull/144/files#diff-774e1282b450306a1e542cc06f593d6d9f995d6753be1d540c10f2de06b4266fR53

similar to how it is used in `testnet_integration.rs` (used it as example for these unit-tests)

---

You could move it under dev-dependencies since its only used for testing

```suggestion
[dev-dependencies]

serde_json = { version = "1.0.67"}
```